examples/chart/teleport-kube-agent/templates/statefulset.yaml

```diff
@@ -32,6 +32,8 @@ spec:
       labels:
         app: {{ .Release.Name }}
     spec:
+      securityContext:
+        fsGroup: 9807
```

Comment:
Mostly because that’s how the original PR was. What we’re doing here is simply setting a consistent GID between the container and the storage - it doesn’t need to be a customisable value, just a consistent one.

Reply:
This is already hardcoded in https://github.com/gravitational/teleport/pull/8085/files#diff-dbbb315f820461e9797bf42f1221f2dd05a4a706b3b1c08540354ee823e9fe77L90 as well.